src/components/payments-network/payments-network.styles.js

```diff
@@ -0,0 +1,136 @@
+import { createUseStyles } from 'react-jss'
+
+const usePaymentsNetworkStyles = createUseStyles(theme => ({
+  section: {
+    padding: '96px 72px',
+    [theme.breakpoints.sm]: {
+      padding: '96px 72px'
+    }
+  },
+  section1: {
+    paddingTop: '86px',
+    backgroundColor: '#f6f7fa',
+    display: 'flex',
+    '& > div': {
+      marginRight: 'auto',
+      marginLeft: 0
+    },
+    '& > div a': {
+      marginLeft: 0
+    }
+  },
+  section2: {
+    '& $col2': {
+      paddingTop: theme.spacing(10),
+      paddingBottom: theme.spacing(10)
+    },
+    '& li': {
+      color: '#667285',
+      fontSize: theme.spacing(2),
+      lineHeight: `${theme.spacing(3.5)}px`,
+      paddingBottom: theme.spacing(1)
+    }
+  },
+  section3: {
+    backgroundColor: '#f6f7fa',
+    textAlign: 'center',
+    '& p': {
+      color: '#778499'
```

Comment:
I think that you can just apply the color in the `section3` class without nesting the `p` element.

Reply:
Actually no, since then the headline will take that colour as well and that one needs to have a different colour.